DB/src/main/java/io/deephaven/db/v2/utils/TableLoggers.java

```diff
@@ -7,6 +7,12 @@
  * Tools to obtain internal, Deephaven logs as tables. These tables include query logs and performance logs.
  */
 public class TableLoggers {
+
+    static {
+        UpdatePerformanceTracker.start();
```

Comment:
Why would we want the UPL on by default, but not the QPL or QOPL?  I thought `ProcessMemoryTracker` was the one we want as a baseline? I would put UPL back the way it was.

Reply:
I didn't think we started those either. I may be mistaken. If someone isn't going to do any performance introspection, we could leave these off by default.

Reply:
See new discussions. I admit that I was definitely wrong.